modules/nw-dns-forward.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * dns/dns-operator.adoc
+
+[id="nw-dns-forward_{context}"]
+= Using DNS forwarding
+
+You can use DNS forwarding to override the forwarding configuration identified in `etc/resolv.conf` on a per-zone basis by specifying which name server should be used for a given zone.
+
+.Procedure
+
+* Modify the DNS Operator object `name: default`. This allows the Operator to create and update the ConfigMap named `dns-default` with additional server configuration blocks based on `Server`. If none of the servers has a zone that matches the query, then name resolution falls back to the name servers that are specified in `/etc/resolv.conf`.
```

Comment:
Sorry if I my earlier comment wasn't clear; there is an object that has the name "default" and the kind "DNS" in the "operator.openshift.io" group.  It can be referenced in `oc` commands as `dnses.operator.openshift.io/default` or just `dnses.operator/default` (either ".operator" or "operator.openshift.io" is needed because there is also a "DNS" kind in the "config.openshift.io" group).  So this line should have ``` named `default` ``` rather than ``` `name: default` ```.

Reply:
Oh, thank you!